libcxx/test/std/algorithms/alg.modifying.operations/alg.fill/ranges.fill_n.pass.cpp

```diff
@@ -48,6 +51,41 @@ constexpr void test_iterators() {
   }
 }
 
+// The `ranges::{fill, fill_n}` algorithms require `vector<bool, Alloc>::iterator` to satisfy the
+// `std::indirectly_writable` concept when used with `vector<bool, Alloc>`, which is only met since C++23.
```

Comment:
```suggestion
// Make sure we behave properly with std::vector<bool> iterators with custom size types, see
// https://github.com/llvm/llvm-project/pull/122410.
//
// The `ranges::{fill, fill_n}` algorithms require `vector<bool, Alloc>::iterator` to satisfy the
// `std::indirectly_writable` concept when used with `vector<bool, Alloc>`, which is only met since C++23.
```